core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataDeleteIndexService.java

```diff
@@ -136,7 +137,8 @@ public void onNodeIndexStoreDeleted(String deleted, String nodeId) {
                 MetaData newMetaData = metaDataBuilder.build();
                 ClusterBlocks blocks = clusterBlocksBuilder.build();
                 RoutingAllocation.Result routingResult = allocationService.reroute(
-                        ClusterState.builder(currentState).routingTable(routingTableBuilder.build()).metaData(newMetaData).build());
+                        ClusterState.builder(currentState).routingTable(routingTableBuilder.build()).metaData(newMetaData).build(),
+                        "reroute after deleting indices [" + indices + "]");
```

Comment:
make that `deleted indices [{}]` (the list can be long pushing the deleted to the end of the line )